examples/nfs/README.md

```diff
@@ -0,0 +1,70 @@
+# Dynamic provisioning with NFS
+
+The easiest way to try out the examples for dynamic provisioning with NFS, is
+to use [Rook Ceph with CephNFS][rook_ceph]. Rook can be used to deploy a Ceph
+cluster. Ceph is able to maintain a NFS-Ganesha service with a few commands,
+making configuring the Ceph cluster a minimal effort.
+
+## Enabling the Ceph NFS-service
```

Comment:
there is no way to enable nfs in standalone steps? do we have any plan to cover standalone examples also?

Reply:
It is possible, but that is not the easiest. Ceph provides the documentation if needed. This README is mainly to get started with the examples, it is not meant to be a exhaustive installation document.